rails/app/assets/stylesheets/cms/_cards.scss

```diff
@@ -117,3 +117,29 @@
     word-wrap: break-word;
   }
 }
+
+.edit-icon-pinned, .edit-icon-not-pinned, .view-icon-pinned {
+  float: right;
+  height: fit-content;
+  margin-inline-start: auto;
+}
+
+.edit-icon-pinned, .view-icon-pinned {
+  color: $black;
+  display: flex;
+}
+
+.edit-icon-not-pinned {
+  color: $light-gray;
+  display: none;
+}
+
+.edit-icon-pinned, .edit-icon-not-pinned {
+  &:hover {
+    box-shadow: 0 0 10px $lighter-gray;
+  }
+}
```

Comment:
Awesome. I just updated a bit because this way the color only changed when the mouse is on icon, but on button doesn't change the color of icon.
I updated to change the color of icon when the mouse is on the button